utils/merkletrie/iter_test.go

```diff
@@ -0,0 +1,462 @@
+package merkletrie_test
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+
+	. "gopkg.in/check.v1"
+
+	"srcd.works/go-git.v4/utils/merkletrie"
+	"srcd.works/go-git.v4/utils/merkletrie/internal/fsnoder"
+	"srcd.works/go-git.v4/utils/merkletrie/noder"
+)
+
+func Test(t *testing.T) { TestingT(t) }
```

Comment:
No it is not, the "main" test file is a test file I will add in a future PR.

Yet, I moved this line here so you can test this part of the code right now, without having to wait for the future PR.